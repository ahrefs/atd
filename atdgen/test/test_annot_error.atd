type pointA <ocaml from="ProtoA"> <ocaml predef from="ProtoB"> <ocaml from="ProtoC" from="ProtoB">= {
  f: float
}
