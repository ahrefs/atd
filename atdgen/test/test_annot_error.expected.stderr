File "test_annot_error.atd", line 1, characters 19-32:
Duplicate annotation ocaml.from (also in:
  File "test_annot_error.atd", line 1, characters 48-61,
  File "test_annot_error.atd", line 1, characters 70-83,
  File "test_annot_error.atd", line 1, characters 84-97
)
